cli/parse-options.js

```diff
@@ -18,6 +19,9 @@ var log = require('npmlog')
  * passe set the `inMemory: true` option. If it’s not set, leveldown is used
  * with the prefix set to `options.data` + 'data' (`.hoodie/data` by default).
  */
+
+PouchDB.plugin(require('pouchdb-mapreduce'))
+
```

Comment:
I followed your instructions up to set the prefix option on the PouchDB. But I have no idea with the after mentions. Do you mean that "move codes in 31-58 from `server/index.js` to 'cli/parse-options.js'?